docs/configuration.md

```diff
@@ -313,6 +315,65 @@ Given the config above, and assuming child config “d” is selected, the follo
 3. The result of 2 will be mixed into the base config
 4. The result of 3 will be the resolved config
 
+## Configuring loaders
+
+A loader is a script used by Intern to load suites into the testing environment. Only a single loader may be specified per
+ environment (Node or browser). Intern has a number of [loaders](https://github.com/theintern/intern/tree/master/src/loaders)
+ available. If no loader is defined, then the default loaders are used per environment.
+
+The Intern's internal loaders may be used simply by specifying their name or location.
+
+```js
+{
+    "browser": {
+        "loader": "dojo"
+    }
+}
+``` 
+ 
+Additional options may be provided through the options parameter. These options are passed through to the registered loader.
+
+```js
+{
+    "browser": {
+        "loader": { 
+            "script": "dojo",
+            "options": { "basePath": '_build' } 
+        }
+    }
+}
+```
+
+It's useful to think of Intern's loaders as glue code that Intern uses to load configured suites within the environment.
+ Consequently, most of the provided Intern loaders also require the loader package to be installed in `node_modules` 
+ using NPM. Look at the [loader code](https://github.com/theintern/intern/tree/master/src/loaders) for more options and
+ requirements.
+
+## Configuring plugins
+
+Plugins are standalone scripts that are loaded by Intern. Any number of them may be loaded in-order following a similar 
```

Comment:
They can also be modules if `useLoader` is true (so maybe "standalone scripts or modules").